### DISCUSSION TITLE
Opening 2 info window

### DISCUSSION THREAD

**Discussion by phoique** [JustFly1984/react-google-maps-api#3082] [Q&A]
Greetings, as you can see in the picture, my markers and when these markers are clicked, I opened the info window, but the info window opens 2 pieces. How can i solve this problem.

Here's how it works. When the marker is clicked, I am sending the lat and long values into the state. If there is lat and long, I show the info window.

[screenshot](https://www.hizliresim.com/1zn7qaz)

**Comment by JustFly1984** [comment]
@phoique did you try InfoWindowF ?

**Reply by phoique** [reply]
Yes this provided a temporary solution. What is the difference between InfoWindow and InfoWindowF?

When I use InfoWindowF, that little info window that is on my screen momentarily opens, and then it is added to the info window that I wrote specifically, and that little bit remains behind. Why this way?

**Reply by JustFly1984** [reply]
The difference is that I've rewritten F components as functional components with hooks to support react@18. 
Info window is separate package.

I'm only maintaining this repo, can't really test, cos I have no project using it, and no api key to test it locally.

I'm waiting for google to provide me with api keys, so I could increase my involvement. Until that I can only accept PRs and release new versions.

Your PR is welcome.
